guides/appendix/live-reload.md

```diff
@@ -0,0 +1,83 @@
+Integrating with ember's live-reload system can be fairly straight-forward.
+
+In short, any tool that can output or change files automatically can be used to add live-reload behavior.
+
+The process is:
+
+- emit some file to either the `public` or `app` tree
+- `link` or `import` the file from existing `ember` code
+
+An example of this behavior and integration can be shown with a CSS authoring tool, Tailwind.
+
+
+[Tailwind](https://tailwindcss.com/) is a popular way to use utility-first classes in projects that helps unify large projects using design-tokens-as-code.
```

Comment:
```suggestion
[Tailwind](https://tailwindcss.com/) is a popular way to use utility-first CSS classes in an app. It can be used within many front end frameworks, including Ember.
```

Reply:
updated